### DISCUSSION TITLE
Manually Running VSCode-Jest for a single file

### DISCUSSION THREAD

**Discussion by komyg** [orta/vscode-jest#701] [Q&A]
Hi,

I am testing this extension to VSCode and I was wondering if I can run jest manually for a single file. I haven't found the option on the documentation, so maybe I am missing something?

Thanks,
Komyg

**Comment by connectdotz** [comment]
hi,

see [how-to-trigger-the-test-run](https://github.com/jest-community/vscode-jest#how-to-trigger-the-test-run), basically you will need to opt-in to the interactive mode to see the context menu and commands for running tests on-demand.
